src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -31,16 +31,14 @@ DateTimeFormatter formatter = DateTimeFormatter.ofLocalizedDateTime(FormatStyle.
   <meta charset="UTF-8">
   <title><%= user.getName() %>'s Profile Page</title>
   <link rel="stylesheet" href="/css/main.css">
-
-  <script>
+</head>
+<script>
```

Comment:
Why did you move this out of the <head>? I was under the impression that a <script> always needed to be within a <head> or a <body>. It looks like it is in neither now.

Reply:
for some reason the first time I updated the appengine, to see if all was working, the scroll function didn't want to work so I moved it around and it was working and I just changed it back right now and it works again? so i'm a bit confused lol

Reply:
Heh.. I thought that might be the case. I definitely recommend keeping it inside the <head> if you can. I'm not exactly sure what might have been going wrong before.